### ISSUE TITLE
scale bar per image type

### ISSUE THREAD

**Issue by joefutrelle** [WHOIGit/photic#208]
Display a scale bar based on which imager it came from (which can be generalized to "image type")

Requires that the model track image type, and that it be specified during import.

**Comment by mike-kaimika** [comment]
@joefutrelle We're looking for something like this image shows, right?
![image](https://user-images.githubusercontent.com/24275288/163730996-0e63c909-597b-4fcc-8a85-b40f763a64f9.png)

**Comment by joefutrelle** [comment]
Yes, and it would need to be scaled when the user zooms in or out on the ROIs
